server/schedulers/hot_region.go

```diff
@@ -229,27 +259,35 @@ func (h *balanceHotRegionsScheduler) balanceHotReadRegions(cluster opt.Cluster)
 const balanceHotRetryLimit = 10
 
 func (h *balanceHotRegionsScheduler) balanceHotWriteRegions(cluster opt.Cluster) []*operator.Operator {
+	balancePeer := h.r.Int()%2 == 0
```

Comment:
why introduce it?

Reply:
Sometimes it will only try one balance strategy. I think that is not what we want.